### ISSUE TITLE
[Bug] Wrong commit message: "$(echo Publish"

### ISSUE THREAD

**Issue by FredyC** [lerna/lerna#271]
I am running `lerna publish` on Windows 10 within Powershell. Apparently it's not possible to use multi line commit message there.

**Comment by dannyfritz** [comment]
I'm getting a similar experience. [This line](https://github.com/lerna/lerna/blob/bfd1fda732ccf7a0b07e6e434fd5575ba862ae2d/src/GitUtilities.js#L23) relies on Bash's [command substitution](https://www.gnu.org/software/bash/manual/html_node/Command-Substitution.html) feature.

lerna: `2.0.0-beta.26`
OS: `Windows 10`
npm: `3.10.4`
node: `6.3.1`

```
? Are you sure you want to publish the above changes? Yes
Errored while running PublishCommand.execute
Error: Command failed: git commit -m "$(echo "Publish

 - fond-debug@1.0.0
 - fond-graphics@1.0.0
 - fond-keyboard@1.0.0
 - fond-runtime@1.0.0
 - fond-timer@1.0.0")"
    at checkExecSyncError (child_process.js:475:13)
    at Object.execSync (child_process.js:515:13)
    at Function.execSync (C:\Program Files\nodejs\node_modules\lerna\lib\ChildProcessUtilities.js:61:40)
    at Function.commit (C:\Program Files\nodejs\node_modules\lerna\lib\GitUtilities.js:77:39)
    at PublishCommand.gitCommitAndTagVersionForUpdates (C:\Program Files\nodejs\node_modules\lerna\lib\commands\PublishCommand.js:388:30)
    at PublishCommand.commitAndTagUpdates (C:\Program Files\nodejs\node_modules\lerna\lib\commands\PublishCommand.js:369:28)
    at PublishCommand.execute (C:\Program Files\nodejs\node_modules\lerna\lib\commands\PublishCommand.js:138:16)
    at PublishCommand._attempt (C:\Program Files\nodejs\node_modules\lerna\lib\Command.js:157:21)
    at C:\Program Files\nodejs\node_modules\lerna\lib\Command.js:142:15
    at C:\Program Files\nodejs\node_modules\lerna\lib\Command.js:166:13

[AMERICANROBIN] C:\Users\Danny\Dev\fond\packages\fond-runtime
```

**Comment by jayphelps** [comment]
Seeing this on Mac OS 10.11.6 using zsh as well as regular old bash-3.2

Anyone know of a workaround? Seems like it's not lack of command substitution support since these shells support it.

**Comment by Gcaufy** [comment]
I got the same issue,
It was pushed successfully. but just get the wrong comment in my commits.
see:
https://github.com/wepyjs/wepy/commit/86cf24f64f116f7edb5b5a6cd47b2656ec29ac52

**Comment by tleunen** [comment]
Having the same issue on MacOS 10.12.2 with zsh as well.

```
GitUtilities.commit           ("Publish\n\n - packageX@1.0.0")
Error: Command failed: git commit -m "$(echo "Publish

 - packageX@1.0.0")"
    at checkExecSyncError (child_process.js:490:13)
    at Object.execSync (child_process.js:530:13)
    at Function.execSync (/Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/ChildProcessUtilities.js:76:40)
    at commit (/Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/GitUtilities.js:81:39)
    at Function.descriptor.value (/Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/logger.js:143:33)
    at PublishCommand.gitCommitAndTagVersionForUpdates (/Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/commands/PublishCommand.js:399:30)
    at PublishCommand.commitAndTagUpdates (/Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/commands/PublishCommand.js:380:28)
    at PublishCommand.execute (/Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/commands/PublishCommand.js:148:16)
    at PublishCommand._attempt (/Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/Command.js:166:21)
    at /Users/tleunen/.nvm/versions/node/v7.2.1/lib/node_modules/lerna/lib/Command.js:151:15
```

**Comment by raoulus** [comment]
Having the same issue when I try to publish a prerelease version (1.0.0-alpha.1). Publishing non prerelease versions (1.0.0) work.

**Comment by Fer22f** [comment]
Any update on this issue? Just got this earlier this morning when trying to publish 0.0.0

**Comment by evocateur** [comment]
This was fixed in #758 (v2.0.0-rc.2)
